pkgs/tools/misc/tere/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec
+  {
```

Please make the formatting look like this (see [here](https://github.com/NixOS/nixpkgs/blob/f768f101891cf5c40601ffeb66342f6478ed1d06/pkgs/tools/text/ripgrep/default.nix) to see how your formatting should look like (2 spaces instead of 4 for the keys)):

```suggestion
rustPlatform.buildRustPackage rec {
```

---

Pushed, is this right?

---

Yup!